torchgeo/models/changestar.py

```diff
@@ -0,0 +1,229 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""ChangeStar implementations."""
+
+from typing import Any, Dict, List
+
+import torch
+from einops import rearrange
+from torch import Tensor
+from torch.nn.modules import (
+    BatchNorm2d,
+    Conv2d,
+    Identity,
+    Module,
+    ReLU,
+    Sequential,
+    UpsamplingBilinear2d,
+)
+
+from .farseg import FarSeg
+
+Identity.__module__ = "nn.Identity"
+
+
+class ChangeMixin(Module):
+    """This module enables any segmentation model to detect binary change.
```

Comment:
Could you clarify this more? Seems like it's just feeding images through conv layers. Particularly the 'enables any segmentation model' part.